core/services/pipeline/task.divide.go

```diff
@@ -0,0 +1,50 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+)
+
+type DivideTask struct {
```

Comment:
We might also consider a "Quotient" task type modelled on the original one in jpv1

Reply:
If I understand that task correctly, you should be able to accomplish that by swapping the variables in the `input` and `divisor` fields

Reply:
Ah, yes! You can.